components/fieldgroup/stories/fieldgroup.stories.js

```diff
@@ -71,17 +71,14 @@ export default {
 		helpText: "Select an option.",
 		items: [
 			{
-				id: "apple",
```

Comment:
these were causing issues with the full docs page when testing with VO since they weren't unique, but the radio template handles for assigning a unique ID